sharepoint/sharepoint-ps/sharepoint-online/New-SPOSiteFileVersionExpirationReportJob.md

```diff
@@ -16,7 +16,7 @@ manager: seanmc
 ## SYNOPSIS
 
 > [!NOTE]
-> This feature is part of the Version history controls Preview. If your tenant is not part of the Preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
 
 Starts generating file version expiration report for a site collection.
```

Comment:
Generate a version storage usage report for a site collection. This report can be used to understand current version storage footprint of the site.

Reply:
I think all these strings use present tense? So "Generates a version..." etc

applies to all files